portfolio/src/main/webapp/scripts/listenersMain.js

```diff
@@ -0,0 +1,9 @@
+const options = document.querySelectorAll(".menu-option");
+const factButton = document.querySelector("#fact-button");
+
+for(let i=0; i<options.length; i++){
```

Comment:
Just a small note.  It is generally considered better style to add spaces around binary operators as it is more readable.  There should also be a space after "for"

i.e.

for (let i = 0; i < options.lenth; i++) {
  ...
}